src/container.c

```diff
@@ -38,19 +40,44 @@ int container_start(void *arg) {
     return -1;
   }
 
+  int exit = 0;
+
+  char **argv = malloc((config->arg_length + 2 * sizeof(char *)));
```

Comment:
On the same line as the comment for the `join_strings` function, this could be something we do in `barco.c` to format user input before passing it to the function to initiliaze/start the container.